pygmt/tests/test_plot.py

```diff
@@ -297,6 +299,25 @@ def test_plot_sizes_colors_transparencies():
     return fig
 
 
+@pytest.mark.mpl_image_compare
+def test_plot_symbol():
+    """
+    Plot the data using array-like symbols.
+    """
+    fig = Figure()
+    fig.plot(
+        x=[1, 2, 3, 4],
+        y=[1, 1, 1, 1],
+        region=[0, 5, 0, 5],
+        projection="X4c",
+        fill="blue",
+        size=[0.1, 0.2, 0.3, 0.4],
+        symbol=["c", "t", "i", "s"],
```

Comment:
Do `size` and `symbol` need to have to same length? Or is something like:

```python
size=[0.1, 0.2, 0.3, 0.4],  # different sizes but same symbol
symbol=["c"],
```
or
```python
size=[0.4],  # same size but different symbols
symbol=["c", "t", "i", "s"],
```
allowed?

Reply:
Currently, they must be the same size.

Reply:
Now I figured it out: One has to use the ``size`` or ``symbol`` parameter together with the ``style`` parameter 🙂.